libexec/install.ps1

```diff
@@ -1,16 +1,48 @@
-# Install miniconda3 if not installed
-$CondaInstallationDirectory = "$Env:userprofile\miniconda3"
+# Set up env variables
+If ($Env:ncDebug){
+    $DebugPreference = "Continue"
+}
+If ($Env:ncCI){
+    Write-Debug "Running inside CI pipeline, turning on non-interactive mode"
+    $Env:ncNoninteractive = $true
+}
+If ($Env:ncNoninteractive){
+    Write-Warning "Running in non-interactive mode - will not prompt for input!"
+}
+If ($Env:ncTargetDirectory){
+    Write-Debug "Found ncTargetDirectory=$Env:ncTargetDirectory"
+    $CondaInstallationDirectory = "$Env:ncTargetDirectory"
+}
+Else {
+    If ($Env:username -match " "){
+        $CondaInstallationDirectory = "$Env:public\miniconda3"
+    }
+    Else {
+        $CondaInstallationDirectory = "$Env:userprofile\miniconda3"
+    }
+}
 $MinicondaLatestUrl = "https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86_64.exe"
-If ($Env:username -match " ") { $CondaInstallationDirectory = "$Env:public\miniconda3" }
 
+function User-Input {
+    $ans = Read-Host "Press RETURN/ENTER to continue or any other key to abort:"
+    if ($ans -eq ""){
```

Comment:
The logic here is inverse to what the prompt says. RETURN/ENTER exists instead of continuing.